client/js/components/map/map/DpOlMapDrawFeature.vue

```diff
@@ -176,12 +177,6 @@ export default {
     }
   },
 
-  watch: {
```

Comment:
Did you make sure that removing this does not break anything in the context it was added (segment Ortsbezug)?

Reply:
This did break segment location map. But the watch created more problems than solving, so now the init of the map is done via event ([8db883e](https://github.com/demos-europe/demosplan-core/pull/3500/commits/8db883e1f553873bb6a973d83510df751de865dc)).